app/tests/spec/lib/validate.js

```diff
@@ -16,16 +16,36 @@ define(function (require, exports, module) {
 
   describe('lib/validate', function () {
     describe('isEmailValid', function () {
+      it('returns true for valid email', function () {
```

Comment:
Not sure if we want to cover this edge case but joi validates emails like `joe@[IPv6:2a00:1450:4001:c02::1b]`. The current regex fails for this.

https://github.com/hapijs/joi/blob/master/test/string.js#L471